webapp/.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-controls",
```

Comment:
I got to know that controls are inherited from `@storybook/addon-essentials`. I think you might as well use those.

Reply:
`addon-essentials` is a meta package including `addon-controls`: https://storybook.js.org/docs/react/essentials/introduction